example/material_stepper_example/lib/material_stepper_demo.html

```diff
@@ -0,0 +1,202 @@
+<!--
+ Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+ for details. All rights reserved. Use of this source code is governed by a
+ BSD-style license that can be found in the LICENSE file.
+-->
+<section>
+  <h2>ACX2 Stepper: horizontal orientation, backwards jumps allowed, default size</h2>
```

Comment:
ACX2?

Reply:
Ha, not a very important description. I'll get those cleaned up.